Update hybrid-agent.md (#3075)
MicrosoftDocs/OfficeDocs-Exchange: Exchange/ExchangeHybrid/hybrid-deployment/hybrid-agent.md

TargetSharingEpr : https://087f1c2e-8711-4176-ab4f-4b1c1777a350.resource.mailboxmigration.his.msappproxy.net/EWS/Exchange.asmx
```

> ## Hybrid Agent PowerShell Module
> The Hybrid Management PowerShell module, installed by the Hybrid Configuration wizard at “\Program Files\Microsoft Hybrid Service\” on the machine where the Hybrid Agent is installed. These module provides following cmdlets which can now be used with or without MFA enabled Administrator account.  

Reviewer: ```suggestion
## Hybrid Agent PowerShell module
The Hybrid Management PowerShell module is installed by the Hybrid Configuration wizard at “\Program Files\Microsoft Hybrid Service\” on the computer where the Hybrid Agent is installed. This module provides the following cmdlets which can now be used with or without a multifactor authentication-enabled administrator account.  
```

|Cmdlets|Purpose|
|---|---|

> |Get-HybridAgent|To view Hybrid Agents Installed |
> |Update-HybridApplication|To edit parameters like Target URI of Hybrid Application |
> |Get-HybridApplication|To view all Hybrid Applications in the Tenant |
> |Remove-HybridApplication|To remove a specific Hybrid Application|

Reviewer: ```suggestion
|Get-HybridAgent|To view Hybrid Agents installed. |
|Update-HybridApplication|To edit parameters like the target URI of Hybrid Application. |
|Get-HybridApplication|To view all Hybrid Applications in the organization.|
|Remove-HybridApplication|To remove a specific Hybrid Application.|
```

> ## How to use these Hybrid Agent Cmdlets with or without MFA enabled accounts
> To use these cmdlets you would need to import the latest HybridManagement.psm1 available at “\Program Files\Microsoft Hybrid Service\”. It can also be downloaded directly from [https://aka.ms/HybridAgentPSM](https://aka.ms/HybridAgentPSM).
> 
> To import the Hybrid Management module, run the following from a Windows PowerShell prompt as Administrator:

Reviewer: ```suggestion
### How to use these Hybrid Agent cmdlets with or without multifactor authentication-enabled accounts
To use these cmdlets you would need to import the latest HybridManagement.psm1 available at **\Program Files\Microsoft Hybrid Service\**. It can also be downloaded directly from [https://aka.ms/HybridAgentPSM](https://aka.ms/HybridAgentPSM).

To import the Hybrid Management module, run the following from a Windows PowerShell prompt as Administrator:
```

```PowerShell
Import-module .\HybridManagement.psm1
```

> Then to execute the above cmdlets with MFA enabled admin credentials you need provide userPrincipalName parameter. If you are using non MFA enabled admin account you can also use Credential parameter in lieu of userPrincipalName parameter which will lead to usage of basic authentication.
> 
> Following table shows how the usage differs in these two modes:

Reviewer: ```suggestion
To execute the these cmdlets with multifactor authentication-enabled admin credentials you need to provide a _userPrincipalName_ parameter. If your account is not a multifactor authentication-enabled admin account, you can also use the _Credential_ parameter in lieu of the _userPrincipalName_ parameter, which will result in the usage of basic authentication.

The following table shows how the usage differs in these two modes:
```

> ||Admin with MFA enabled account|Admin with Basic Auth|

Reviewer: ```suggestion
||Admin with multifactor authentication-enabled account|Admin with basic authentication|
```
